### ISSUE TITLE
`loss` is different after saving and then loading a model

### ISSUE THREAD

**Issue by fzhang612** [bnosac/ruimtehol#8]
I trained a model with `loss = 'softmax'` and saved it into a binary file by `starspace_save_model`. Later on I used `starspace_load_model` to load the same binary file and realized its `loss` became 'hinge'. Thanks!

**Comment by jwijffels** [comment]
you should use starspace_load_model with type 'ruimtehol', otherwise you need to pass on all training parameters again. can you try

**Comment by fzhang612** [comment]
If a model is saved and then loaded back with `method = 'binary'`, its hyperparameters are lost.

If with `method = 'ruimtehol'`, all hyperparameters are preserved.

Didn't test the other 2 options.

Thanks!

**Comment by jwijffels** [comment]
Closing as the loading using method = 'binary' and method = 'tsv-starspace' has been disabled since commit https://github.com/bnosac/ruimtehol/commit/9791b1a674d3dcf37796aef85dac222f3eb9944b.
You should now use the method 'ruimtehol', which is now the default.
